test/java/src/com/ibm/streamsx/topology/test/splpy/PythonFunctionalOperatorsTest.java

```diff
@@ -62,7 +60,13 @@
     		  "map<float64,int32> mf64i32," +
     		  "map<float64,uint32> mf64u32," +
     		  "map<float64,rstring> mf64r," +
-    		  "map<rstring,float64> mrf64>");
+    		  "map<rstring,float64> mrf64";
+
+    public static final StreamSchema ALL_PYTHON_TYPES_SCHEMA =
+            Type.Factory.getStreamSchema(ALL_PYTHON_TYPES_STRING1+">");
+
+    public static final StreamSchema ALL_PYTHON_TYPES_WITH_SETS_SCHEMA =
```

Comment:
FYI - there's a cleaner way to do this.

A schema can be extended using the extend() methods, so it's better to create the first schema and then use it to add the extra attribute with extend.